qgis.dtd

```diff
@@ -22,7 +22,6 @@
 	projectname CDATA #REQUIRED 
 	mapunits (meters|feet|degrees) "meters"
   >
-<!ATTLIST projectlayers layercount CDATA #REQUIRED >
```

Comment:
It hasn't been touched in 13 years apparently, I'm not even sure if we should keep this at all, but let's wait for 3.0 before removing it.